lib/iris/coord_systems.py

```diff
@@ -914,11 +914,11 @@ def __repr__(self):
         return ("LambertAzimuthalEqualArea(latitude_of_projection_origin={!r},"
                 " longitude_of_projection_origin={!r}, false_easting={!r},"
                 " false_northing={!r}, ellipsoid={!r})").format(
-                   self.latitude_of_projection_origin,
-                   self.longitude_of_projection_origin,
-                   self.false_easting,
-                   self.false_northing,
-                   self.ellipsoid)
+                    self.latitude_of_projection_origin,
+                    self.longitude_of_projection_origin,
+                    self.false_easting,
+                    self.false_northing,
+                    self.ellipsoid)
```

Comment:
This change, though unrelated to my own was due to a pep8 failure at my end (perhaps I was using a different pep8 version...).  In any case, this simple corrects the indentation from 3 spaces to 4 spaces.